packages/components/rollup.config.mjs

```diff
@@ -31,16 +31,6 @@ const plugins = [
     'helpers/**/*.js',
     'modifiers/**/*.js',
   ]),
-  // This babel config should *not* apply presets or compile away ES modules.
```

Comment:
I can move this to a separate PR but it would need to be merged before this one (without it I was seeing this build error https://github.com/hashicorp/design-system/actions/runs/8238195028/job/22528807863)

Reply:
Resolving as this was reverted!